stubs/fpdf2/fpdf/syntax.pyi

```diff
@@ -17,42 +17,47 @@ def create_list_string(list_): ...
 def iobj_ref(n): ...
 def create_stream(stream): ...
 
+class Raw(str): ...
+
+class Name(str):
+    NAME_ESC: ClassVar[Pattern[bytes]]
+    def serialize(self) -> str: ...
+
 class PDFObject:
-    def __init__(self, id: Any | None = ...) -> None: ...
+    def __init__(self) -> None: ...
     @property
-    def id(self): ...
+    def id(self) -> int: ...
     @id.setter
-    def id(self, n) -> None: ...
+    def id(self, n: int) -> None: ...
     @property
-    def ref(self): ...
-    def serialize(self, fpdf: Any | None = ..., obj_dict: Any | None = ...): ...
+    def ref(self) -> str: ...
+    def serialize(self, obj_dict: Incomplete | None = ...) -> str: ...
+    def content_stream(self) -> bytes: ...
+
+class PDFContentStream(PDFObject):
+    filter: Name | None
+    length: int
+    def __init__(self, contents: bytes, compress: bool = ...) -> None: ...
```

Comment:
Only if `compress` is `True`. Otherwise it gets assigned to `_contents`, which will eventually be returned by `PDFObject.content_stream`. We could add an overload here, but I think this would be overkill.